tests/python/direct/test_python_frontend.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# Owner(s): ["module: nvfuser"]
+
+import pytest
+import torch
+import torch._refs as refs
+import torch._prims as prims
+
+from nvfuser_direct import (
+    FusionDefinition,
+    DataType,
+)
+
+
+def test_basic(nvfuser_direct_test):
```

Comment:
Switched to a `pytest fixture` instead of a large, monolithic class for flexibility.
Do you want to do the same for legacy `test_python_frontend.py`?